### DISCUSSION TITLE
I'm having problems with Extended Color Schemes :(

### DISCUSSION THREAD

**Discussion by aryawithay** [AnubisNekhet/AnuPpuccin#122] [Snippets]
I'm unable to use the extended color schemes snippet through style settings. I can download, add, and enable it but it doesn't show up at all in the Style Settings option page. This happens regardless of whether the vault is brand new with no other plugins or if it is old and has many community plugins. Is this a problem anyone else has had and are there any solutions to this? Thank you.

**Comment by ms3056** [comment]
Hmmm - no issue for me. Just to be sure, screenshots of the settings. 
![image](https://user-images.githubusercontent.com/23712700/225174865-25de509e-335d-4c07-b741-4d7c1f3b1403.png)

**Reply by Practical-Seaweed** [reply]
how do you have the extended-colorschemes in the appearance section?

**Reply by ms3056** [reply]
Install the extended-colors schemes.css snippet from the snippets repository and turn it on. You will then see the extended colors in Style Settings.
